source/agora/consensus/data/genesis/Coinnet.d

```diff
@@ -60,42 +60,42 @@ unittest
 /// The enrollments that are part of the genesis block
```

Comment:
What is the condition that it's asserting though? assert they're sorted? or something else? It should be in the commit message & function doc header.